packages/browser/src/sessionInfo/SessionInfoManager.ts

```diff
@@ -170,4 +170,27 @@ export class SessionInfoManager implements ISessionInfoManager {
   async clear(sessionId: string): Promise<void> {
     return clear(sessionId, this.storageUtility);
   }
+
+  /**
+   * Registers a new session, so that its ID can be retrieved.
+   * @param sessionId
+   */
+  async register(_sessionId: string): Promise<void> {
+    throw new Error("Unimplemented");
```

Comment:
Is this because of a shared interface for something that we don't want/need to implement in the browser?

If so: ugh... Please can we just split the two projects? :P

Reply:
I'm afraid so ^^.